tests/FileRemovalTest.php

```diff
@@ -127,6 +127,8 @@ public function testWakeup(): void
      *
      * @runInSeparateProcess
      *
+     * @preserveGlobalState disabled
```

Comment:
intrested that it was not needed so far

Reply:
I wouldn't call it "interesting", more like hidden "gem" of PHPUnit, I've extracted it in https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7579#issuecomment-1864910163